projects/core/src/cms/facade/page-meta.service.ts

```diff
@@ -22,13 +30,15 @@ export class PageMetaService {
 
   constructor(
     protected cms: CmsService,
-    protected unifiedInjector?: UnifiedInjector
+    protected unifiedInjector?: UnifiedInjector,
+    @Optional() protected config?: CmsConfig,
+    @Optional() @Inject(PLATFORM_ID) protected platformId?: string
   ) {}
+
   /**
    * The list of resolver interfaces will be evaluated for the pageResolvers.
-   *
-   * TODO: optimize browser vs SSR resolvers; image, robots and description
-   *       aren't needed during browsing.
+   * This list is filtered for CSR vs SSR processing since not all resolvers are
+   * relevant during browsing.
    */
   protected resolverMethods: { [key: string]: string } = {
     title: 'resolveTitle',
```

Comment:
I recall **a year ago** we had a pending idea to make the mapping configurable (i.e. title -> resolveTitle). Does it still feel interesting idea for you?

https://github.com/SAP/spartacus/blob/5097a6a6ce021e6c7f8cd57ac140cce763da2c8c/projects/core/src/cms/facade/page-meta.service.ts#L27

**Pros**: easier way of adding a new page meta by a customer or a plugin author. Currently multiple sources cannot contribute to one `protected` property, they just overwrite. But how likely it is we'll have many contributors to this value?

**Cons**: it's harder to opt out from some mapping (beacause of deep merge on config chunks). But is it a real case someone would opt out? Would it be a pain to could opt-out by `null`/`undefined`?

If the idea of having a config for the mapping is still interesting for you, we could also control there if the page meta is disabled in CSR, i.e. :

```ts
seo.pageMetaResolvers.robots = {
  key: 'resolveRobots', 
  disableCSR?: true
}
```

Reply:
I've followed this idea and added it to the new pageMeta config.